astronomer/providers/amazon/aws/xcom_backends/s3.py

```diff
@@ -0,0 +1,66 @@
+import json
+import os
+import uuid
+from typing import Any, Union
+
+import pandas as pd
+from airflow.models.xcom import BaseXCom
+from airflow.providers.amazon.aws.hooks.s3 import S3Hook
+
+
+class S3XComBackend(BaseXCom):
+    """
+    Custom XCom persistence class extends base to support various datatypes.
+    To use this XCom Backend, add the environment variable `AIRFLOW__CORE__XCOM_BACKEND`
+    to your environment and set it to
+    `astronomer.providers.amazon.aws.xcom_backends.s3.S3XComBackend`
+    """
+
+    PREFIX = os.getenv("PREFIX", "s3_xcom_")
+    AWS_CONN_ID = os.getenv("CONNECTION_NAME", "aws_default")
+    BUCKET_NAME = os.getenv("XCOM_BACKEND_BUCKET_NAME", "airflow_xcom_backend_default_bucket")
+
+    @staticmethod
+    def write_and_upload_value(value: Any) -> str:
+        """Convert to string and upload to GCS"""
```

Comment:
"""Convert to string and upload to GCS""".  It is mentioned as GCS, it has to be S3 right ?

Reply:
changed